lib/buildkite/test_collector.rb

```diff
@@ -39,15 +39,17 @@ class << self
       attr_accessor :session
       attr_accessor :tracing_enabled
       attr_accessor :artifact_path
+      attr_accessor :failures_only
       attr_accessor :env
       attr_accessor :batch_size
     end
 
-    def self.configure(hook:, token: nil, url: nil, tracing_enabled: true, artifact_path: nil, env: {})
+    def self.configure(hook:, token: nil, url: nil, tracing_enabled: true, artifact_path: nil, failures_only: false, env: {})
```

Comment:
I don't know if adding the arg in the middle of the sequence will cause failures, but the readme says to use named args, so it should be fine?

Reply:
That should be fine, I don't believe there's any way any caller could be using this method in an order-dependent way 👍🏼